core/config/initializers/assets.rb

```diff
@@ -1 +1,3 @@
-Rails.application.config.assets.precompile << 'solidus_core_manifest.js'
+if Rails.application.config.respond_to?(:assets)
```

Comment:
I think we should remove this changes since are in https://github.com/solidusio/solidus/pull/2358

Reply:
This is built on top of #2358, which is necessary for this to pass.